### PR TITLE
release: v1.39.2

### PR DESCRIPTION
**Pull request by gsquared94**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.39.2`
+
+Note: This release is a patch to v1.39.1, fixing an `index out of range error` caused when rebuilding an artifact with a new tag [#7587].
+
+Fixes:
+* fix: index out of range error [#7593](https://github.com/GoogleContainerTools/skaffold/pull/7593)
+
 # v1.39.1 Release - 06/28/2022
```
Comment (aaron-prindle) [inline CHANGELOG.md:25]: nit: should we remove the changelog note that this was fixed from `v1.39.1` as well then?

**Review by aaron-prindle — APPROVED** [review]
LGTM

**Comment by aaron-prindle** [comment]
Also, please submit another PR adding this to the `main` Changelog as well.  Thanks

File: CHANGELOG.md (hunk 1 above)
Comment (gsquared94) [reply CHANGELOG.md:25]: done

Comment (aaron-prindle) [reply CHANGELOG.md:25]: nit: might also want to edit our Github releases and change it there as well

**Pull request merged**